lib/Twig/Extension/HelpersExtension.php

```diff
@@ -18,13 +18,38 @@
 namespace Pimcore\Twig\Extension;
 
 use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+use Twig\TwigFunction;
 use Twig\TwigTest;
 
 /**
  * Simple helpers that do not need a dedicated extension
  */
 class HelpersExtension extends AbstractExtension
 {
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('basename', [$this, 'basenameFilter']),
+        ];
+    }
+
+    public function getFunctions()
+    {
+        return [
+            new TwigFunction('callStatic', function ($class, $method, $args = array()) {
```

Comment:
not a good idea I think. This seems to a bad practice. Better create specific twig extensions where this is needed.

Reply:
also, naming scheme is not persisted, use camel case and prefix with pimcore_